ci/release/update-version.sh

```diff
@@ -45,3 +45,7 @@ sed_runner 's/'"branch-.*\/RAPIDS.cmake"'/'"branch-${NEXT_SHORT_TAG}\/RAPIDS.cma
 for FILE in conda/environments/*.yml; do
   sed_runner "s/cudf=${CURRENT_SHORT_TAG}/cudf=${NEXT_SHORT_TAG}/g" ${FILE};
 done
+
+# Doxyfile update
+sed_runner "/PROJECT_NUMBER/ s|[0-9]\+.[0-9]\+|${NEXT_SHORT_TAG}|g" cpp/doxygen/Doxyfile
```

Comment:
```suggestion
sed_runner "/PROJECT_NUMBER[ ]*=/ s|=.*|= ${NEXT_FULL_TAG}|g" cpp/doxygen/Doxyfile
```
One more update needed here. You can test this with various version numbers to ensure that everything works as intended. For example, run the following and check the `diff` each time.

```sh
bash ci/release/update-version.sh 22.12.00
bash ci/release/update-version.sh 22.12.02
```